### PR TITLE
Feature/check in

### PR DESCRIPTION
**Pull request by iamgenechua**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/CheckInCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                PREFIX_START_DATE, PREFIX_END_DATE);
+
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_PERSONAL_ID, PREFIX_ROOM_ID, PREFIX_START_DATE, PREFIX_END_DATE)
+            || !argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, CheckInCommand.MESSAGE_USAGE));
+        }
+
+        int personalId = ParserUtil.parsePersonalId(argMultimap.getValue(PREFIX_PERSONAL_ID).get());
+        int roomId = ParserUtil.parseRoomId(argMultimap.getValue(PREFIX_ROOM_ID).get());
```
Comment (Hazel1603) [inline src/main/java/seedu/address/logic/parser/CheckInCommandParser.java:37]: Just want to double check, the checking of the length of the roomId is done in the execute which is not yet implemented right?

**Review by Hazel1603 — APPROVED** [review]
LGTM!

**Review by shermz-lim — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/parser/CheckInCommandParser.java (hunk 1 above)
Comment (iamgenechua) [reply src/main/java/seedu/address/logic/parser/CheckInCommandParser.java:37]: as what @shermz-lim mentioned earlier during our meeting, he suggested that there shouldn't be a checking of length for the room id. if the user keys in roomId of length of 5, for example, the user will just be greeted with a 'room does not exist'

**Pull request merged**